app/api/youtube/channel-info/route.ts

```diff
@@ -0,0 +1,52 @@
+import { NextRequest, NextResponse } from "next/server";
+import { fetchYouTubeChannelInfo } from "@/lib/youtube/channel-fetcher";
+import { validateYouTubeTokens } from "@/lib/youtube/token-validator";
+import getAccountArtistIds from "@/lib/supabase/accountArtistIds/getAccountArtistIds";
+
+export async function GET(request: NextRequest) {
+  try {
+    const { searchParams } = new URL(request.url);
+    const artistAccountId = searchParams.get("artistAccountId");
+    const accountId = searchParams.get("accountId");
+    const includeBranding = searchParams.get("include_branding") === "true";
```

Comment:
YAGNI
- is this param include_branding ever being used? if no, it looks like it's not, please remove it